editor/plugins/script_editor_plugin.cpp

```diff
@@ -2255,11 +2255,17 @@ bool ScriptEditor::edit(const Ref<Resource> &p_resource, int p_line, int p_col,
 		if (!has_file_flag) {
 			args.push_back(script_path);
 		}
+        
+        if (path.is_empty()) {
+            ERR_PRINT("use_external_editor flag is set, but no path to external text editor is set")
+            return false;
+        } else {
```

Comment:
You can use instead (also fixed formatting):
```suggestion
        ERR_FAIL_COND_V_MSG(path.is_empty(), false, "The `use_external_editor` setting is enabled, but the path to the external text editor is not configured in `text_editor/external/exec_path`.");
```

The indentation also needs to be fixed (you used spaces when we use tabs everywhere).

Reply:
That being said, I think erroring it and aborting is not the right solution here. Just below there is `WARN_PRINT("Couldn't open external text editor, using internal");` (which could get an extra period at the end btw).

I think this is sufficient, albeit it could be turned into an `ERR_PRINT`, maybe with slightly improved text. So you should just skip creating the process if there's no path.

```
if (!path.is_empty()) {
	Error err = OS::get_singleton()->create_process(path, args);
	if (err == OK) {
		return false;
	}
}

ERR_PRINT("Couldn't open external text editor, falling back to the internal editor. Review your `text_editor/external/` editor settings.");
```